### ISSUE TITLE
Ability to #createSvg on the WebApp for vector marker icons

### ISSUE THREAD

**Issue by WiIIiam278** [BlueMap-Minecraft/BlueMapAPI#3]
Hi there,
I'd like to add some custom markers to display in my hook.
One of the neat things BlueMap supports is SVG graphics as marker icons (great!). Unfortunately, `WebApp#createImage` only accepts BufferedImages -- so there's no easy way to use vector marker icons on the map.
I'd like for a way to be able to do the same but for (.svg) vector graphics if possible.

Apologies for the slightly specific request -- and thanks for your work on API v2, it's great to work with!

**Comment by TBlueF** [comment]
I agree, there should be a way..
I am generally not pleased with the way the resource-management works right now.. and i might change it in a similar way that markers work.
So going away from saving static resources into the webroot and towards providing resources on storage-level on a per-map basis..
Since the current way is also not really compatible with how bluemap works in server-networks.

**tl;dr:** It might not be a second "createImage" function, but i want to add a way for plugins to provide SVG-files and more :)

**Comment by TBlueF** [comment]
Implemented
